packages/components/modal/src/drawer.tsx

```diff
@@ -89,7 +96,7 @@ export const Drawer = motionForwardRef<DrawerProps, "div">(
       placement,
       ...props,
     })
-    const {
+    let {
```

Comment:
Please define the variable as const, similar to how it's done with`packages/components/modal/src/drawer-content.tsx` .